routes/reviews-router.js

```diff
@@ -3,9 +3,15 @@ const { getReviews, getReviewById, getCommentsByReviewId, postComment, patchVote
 
 reviewsRouter
     .route("/")
-    .get((req, res) => {
-        getReviews().then((reviews) => {res.status(200).send({ reviews })})
-    })
+    .get((req, res, next) => {
+        const { category, sort_by, order } = req.query; 
+        getReviews(category, sort_by, order)
+        .then((reviews) => {
+            //console.log(reviews)
+            res.status(200).send({ reviews })
+        })
+        .catch(next);
+    });
```

Comment:
This should be extracted out to a controller file. The same should be done for your other routers if they are done in the same way.